command/identify.py

```diff
@@ -0,0 +1,13 @@
+import yaml
+
+def getOperatorArtifactType(operatorArtifactString):
```

Comment:
This is not a pythonic name, please see PEP8 https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Addressed in the next commit.